### PR TITLE
Update build

### PR DESCRIPTION
**Pull request by axuan25**

### CODE REVIEW

**Review by nbrr — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ version: 2.1
 
 defaults: &defaults
   docker:
-    - image: circleci/android:api-29
+    - image: circleci/android:api-31
```
Comment (nbrr) [inline .circleci/config.yml:9]:
```suggestion
    - image: cimg/android:2021.12.2
```

See https://circleci.com/docs/2.0/circleci-images/

Comment (axuan25) [reply .circleci/config.yml:9]: Thanks! Moving a little too quickly 😄

File: sdk-facebook/src/main/java/co/reachfive/identity/sdk/facebook/FacebookProvider.kt [hunk 2]
```diff
@@ -67,42 +63,34 @@ class ConfiguredFacebookProvider(
...
-                    val accessToken = result?.accessToken?.token
-                    if (accessToken != null) {
-                        val loginProviderRequest = LoginProviderRequest(
-                            provider = name,
-                            providerToken = accessToken,
-                            clientId = sdkConfig.clientId,
-                            origin = origin,
-                            scope = scope.joinToString(" ")
+                override fun onSuccess(result: LoginResult) {
+                    val accessToken = result.accessToken.token
```
Comment (StanWarz) [inline sdk-facebook/src/main/java/co/reachfive/identity/sdk/facebook/FacebookProvider.kt:67]: Ce cas d'erreur ne se présente jamais ?

File: sdk-webview/src/main/java/co/reachfive/identity/sdk/webview/ReachFiveLoginActivity.kt [hunk 3]
```diff
@@ -48,7 +48,7 @@ class ReachFiveLoginActivity : Activity() {
 
         Log.d(TAG, "ReachFiveLoginActivity onCreate : $url")
 
-        webview.loadUrl(url)
+        url?.let { webview.loadUrl(it) }
```
Comment (StanWarz) [inline sdk-webview/src/main/java/co/reachfive/identity/sdk/webview/ReachFiveLoginActivity.kt:51]: 👍

**Review by StanWarz — APPROVED** [review]

File: sdk-facebook/src/main/java/co/reachfive/identity/sdk/facebook/FacebookProvider.kt (hunk 2 above)
Comment (axuan25) [reply sdk-facebook/src/main/java/co/reachfive/identity/sdk/facebook/FacebookProvider.kt:67]: Discussed: the returned values are no longer nullable ✔️

**Pull request merged**